src/rust/grapl-utils/src/rusoto_ext/dynamodb.rs

```diff
@@ -0,0 +1,178 @@
+use std::collections::HashMap;
+
+use async_trait::async_trait;
+use rusoto_core::RusotoError;
+use rusoto_dynamodb::{AttributeValue,
+                      BatchGetItemError,
+                      BatchGetItemInput,
+                      BatchGetItemOutput,
+                      DynamoDb,
+                      KeysAndAttributes};
+
+const DYNAMODB_MAX_BATCH_GET_ITEM_SIZE: usize = 100;
+
+#[async_trait]
+pub trait GraplDynamoDbClientExt: DynamoDb {
```

Comment:
Possible to get some metrics in here?

Reply:
I would, but I was really hoping to keep this as a drop-in replacement for `batch_get_item` (so, I'd like to avoid adding additional arguments and modifying the return type if possible).

Let me know what you think. Thanks!

Reply:
Yeah I think that's fair for now.